### DISCUSSION TITLE
Funny AI mistakes

### DISCUSSION THREAD

**Discussion by tubalainen** [blakeblackshear/frigate#864] [Show and tell]
Darn you Frigate! This is beyond insulting for 40+ people with questionable hair thickness! 😆 

![image](https://user-images.githubusercontent.com/26795429/109496454-f806b300-7a90-11eb-88df-6c04dcd3154b.png)

**Comment by Trashmee** [comment]
:-) That looks soooo familar! 61% cat.

![image](https://user-images.githubusercontent.com/8881956/109558898-42f8e880-7ada-11eb-96be-1cadc4263e6f.png)

**Comment by Trashmee** [comment]
But.. this one is 60% cat - not funny, is it..?

![image](https://user-images.githubusercontent.com/8881956/109559062-818ea300-7ada-11eb-98a1-af1b862d3063.png)

**Reply by tubalainen** [reply]
Eh ... no ... eh yea! uh? ....

**Reply by jakekeeys** [reply]
i see it :joy: ![signal-2021-05-30-100345](https://user-images.githubusercontent.com/22008619/120098549-73c6d900-c12e-11eb-970c-2072a45606f7.png)

**Comment by natm** [comment]
Frigate reminding me to wind up the garden hose pipe!

![EsMAdLnWMAIoPUK](https://user-images.githubusercontent.com/1790618/110257962-21937300-7f98-11eb-84d9-6ed2333057cb.png)

**Reply by tubalainen** [reply]
Huhum thats a good one! :)

**Comment by tubalainen** [comment]
Just got an alert that there is a person on the front patio..... No, Frigate my dear, its a WEBER GRILL! :)
![image](https://user-images.githubusercontent.com/26795429/114391448-cad11880-9b97-11eb-9c20-2a078a3f0d2a.png)

**Reply by beetlezap** [reply]
Does the same with my Weber... ;)

**Comment by drthanwho** [comment]
My first bird!
![image](https://user-images.githubusercontent.com/22405505/120077657-c27f5f00-c0ab-11eb-8ac0-a13f010892ef.png)

**Comment by jakekeeys** [comment]
![2021-05-29-210233_485x417_scrot](https://user-images.githubusercontent.com/22008619/120083531-35470500-c0c1-11eb-8abc-00685742f3fb.png)

**Comment by jakekeeys** [comment]
![2021-05-30-094623_392x433_scrot](https://user-images.githubusercontent.com/22008619/120098111-f306dd80-c12b-11eb-8b09-4724ed5dc57a.png)

![image](https://user-images.githubusercontent.com/22008619/120098076-ceab0100-c12b-11eb-845d-8869c2328471.png)
got a few laughs from telling the neighbours their dog had escaped and sending them this ^

can't add the sheep models because this guy
![2021-05-30-094930_361x266_scrot](https://user-images.githubusercontent.com/22008619/120098185-5db81900-c12c-11eb-8fe9-9a8a58dae353.png)

**Comment by fti7** [comment]
Ghosts?
![image](https://user-images.githubusercontent.com/7014664/122439218-bea07780-cf9b-11eb-9712-b6ad2d11378f.png)

**Comment by digiblur** [comment]
The long hair humans called dogs is interesting...haven't shown the wife that one yet ;)

**Comment by Kate222** [comment]
My husband had been identified as a cat in one of the event photos. I just could not stop laughing when I saw it.

**Comment by mattheys** [comment]
I had people detected as horses for a while, in the end I took that one off as it never detected the actual horses that went by as they were too far away.

**Comment by RichardPar** [comment]
Keeping all the options open... eventually one will be right

![options](https://user-images.githubusercontent.com/1158641/126032844-d996bb39-a156-4ba2-b4d5-de4221a8beef.png)

**Comment by tubalainen** [comment]
Well ... The "Person" part works great.
But it so-so with some of the other labels. :)
![image](https://user-images.githubusercontent.com/26795429/129247122-e7c9584c-7ca2-4a82-8857-ef31cad0076f.png)

**Reply by therealcarnivor** [reply]
![image](https://user-images.githubusercontent.com/46380907/129263247-8fbbde2b-d767-4a61-852d-fa142ab050e3.png)
questionable....

**Reply by tubalainen** [reply]
Well, 55% confidence, so thats not too bad I´d say. I need 70% or more for my system to trigger on "Person" :)

**Comment by tubalainen** [comment]
"Dog" and "Sheep" works great on Deer
"Dog" works wonders on Badger
"Horse" and "Cow" works great on Moose - here is a picture of a 82% confidence "Horse" 

We soon have the most common Scandinavian wild life mapped 😆 
![snapshot-1630434316 712235-sdv5pu](https://user-images.githubusercontent.com/26795429/131622387-945325d7-7c4d-4f65-9add-a48c25b7b0e1.jpg)

**Comment by mmaibaum** [comment]
I reliably get a few of these a week 
<img width="1103" alt="Screenshot 2021-09-06 at 08 34 30" src="https://user-images.githubusercontent.com/900271/132178051-a78db232-1530-4428-a4fd-fc93dd946535.png">

**Comment by mattheys** [comment]
Does Frigate now have an xray mode?

![garage_bird_1602113099175](https://user-images.githubusercontent.com/10292436/137882188-777c4dc2-f625-4b22-adab-4f9c363ec39b.jpg)

**Comment by tlundste** [comment]
Um... no.
![image](https://user-images.githubusercontent.com/12734342/139956744-cffdedc6-9a43-4eb6-ab0f-39d92315ae1b.png)

**Comment by stu1811** [comment]
![image](https://user-images.githubusercontent.com/12236842/140128323-3613cd0a-2efe-4d24-9edb-af3434f8231b.png)

**Comment by stu1811** [comment]
![image](https://user-images.githubusercontent.com/12236842/140394856-8d1ea096-ad5e-467f-9032-8428b0130445.png)

**Comment by jkarmfalk** [comment]
![254090565_584485626098281_5779746200354291383_n](https://user-images.githubusercontent.com/72249179/141268176-db7f983c-bda0-4aee-81e9-d4b3a7facb69.jpg)

**Comment by mbirrell66** [comment]
![Kangaroo](https://user-images.githubusercontent.com/38741275/142796356-edd2933b-7031-4445-adf0-2021fba4a90d.png)
yep, there is a dog at my gate!

**Reply by mpfl** [reply]
Getting Skippy mixed up with Charlie the wonder dog.

**Reply by scooper1** [reply]
My golden retriever named  Skippy.... Gets identified as a cat

**Comment by n0rthking** [comment]
![dog](https://user-images.githubusercontent.com/93261911/142845209-bcfd3b58-31ea-4e4e-802b-1e20e0dc9090.jpg)
![person](https://user-images.githubusercontent.com/93261911/142845237-261e7b4a-3340-433a-be3b-dff57d6d5a0a.jpg)

**Comment by tubalainen** [comment]
The snowman ... :)
![image](https://user-images.githubusercontent.com/26795429/148655961-07d46001-9f56-4447-8734-343511ce66d3.png)

**Comment by etamtlosz** [comment]
![image](https://user-images.githubusercontent.com/7436784/149063527-88f7e376-aaac-46b8-ba88-6c6de6478089.png)
Where's the person?
Fun fact, frigate always detects the side of my car as a person.

**Comment by etamtlosz** [comment]
And another "person"
![image](https://user-images.githubusercontent.com/7436784/151341738-c73a14d0-364f-4cc1-93fc-87add31c7bff.png)

**Comment by Firerouge** [comment]
During the holidays there was a wreath that frigate would constantly say up to 83% certainty was a person.
![thumbnail (3)](https://user-images.githubusercontent.com/9072079/151688251-4e59bd93-7142-4987-bc7b-9a7cf790547a.jpeg)

**Comment by itkfilelor** [comment]
![Not_A_Dog](https://user-images.githubusercontent.com/26335519/236593157-dd6c249c-669c-43a6-8934-eb8ae3dc4860.png)

She's *NOT* gonna be happy if she sees this!
